backend/Recodo.API/Controllers/MailController.cs

```diff
@@ -0,0 +1,33 @@
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Recodo.BLL.Services;
+using Recodo.Common.Dtos.Video;
+using System.Threading.Tasks;
+using Recodo.API.Extensions;
+
+namespace Recodo.API.Controllers
+{
+    [Route("api/[controller]")]
+    [Authorize]
+    [ApiController]
+    public class MailController : ControllerBase
+    {
+        private readonly MailService _mailService;
+        private readonly UserService _userService;
+
+        public MailController(MailService mailService, UserService userService)
+        {
+            _mailService = mailService;
+            _userService = userService;
+        }
+
+        [HttpPost]
+        [AllowAnonymous]
```

Comment:
What's the point?

Reply:
I don`t remember, but i don`t think it`s needed